content/guides/virtualizing/simnow.md

```diff
@@ -5,42 +5,104 @@ date = "2016-12-23T06:09:57.000Z"
 tags = []
 +++
 
-<div class="box-info">This article is aimed at developers, as AMD SimNow is both a Virtual machine and an AMD hardware emulator. If you just want to try Haiku out, you might want to use another VM that is more aimed at end users.</div>
+This article is aimed at developers, as AMD SimNow is both a Virtual machine and an AMD hardware emulator. If you just want to try Haiku out, you might want to use another VM that is more aimed at end users.
 
 ##### Go to section:
 
-*   [Downloading Haiku.](#part_download-h)
-*   [Downloading AMD SimNow.](#part_download-s)
-*   [Installing AMD SimNow.](#part_installing-s)
-*   [Preparing the Virtual Machine.](#part_preparing-vm)
-*   [Running the Virtual Machine.](#part_running-vm)
+*   [Downloading Haiku](#part_download-h)
+*   [Downloading AMD SimNow](#part_download-s)
+*   [Installing AMD SimNow](#part_installing-s)
+*   [Preparing the Virtual Machine](#part_preparing-vm)
+*   [Running the Virtual Machine](#part_running-vm)
 
-### Downloading Haiku
+
+### <a name="part_download-h"></a> Downloading Haiku 
 
 There are several methods offered to install Haiku. For AMD SimNow will need to download either the iso image or the anyboot image [here](http://www.haiku-os.org/get-haiku) which is a mix of the iso image and the RAW image and can directly be used as an HDD, i.e. it does not require installation.
```

Comment:
There are only Anyboot images and RAW images for download. So you can remove the "either the iso image or" and change to "which is a mix of an ISO image and a RAW image" later in the sentence.

Reply:
Should I also change the link to https://download.haiku-os.org/, which contains nightly builds?

Reply:
No, after the next release "get-haiku" is the right place to go. And the nightlies are linked from there until then, too.